website/docs/integrations/export-evaluation-data/opentelemetry.mdx

```diff
@@ -0,0 +1,123 @@
+````mdx
```

Comment:
The whole page is wrapped in a ```mdx code block.

The generated page show the source markdown instead of the page.
See here: https://deploy-preview-4549--go-feature-flag-doc-preview.netlify.app/docs/next/integrations/export-evaluation-data/opentelemetry